docs/conf.py

```diff
@@ -383,7 +383,7 @@ def linkcode_resolve(domain, info):
   "jax-101/06-parallelism.md": "sharded-computation.md",
   "jax-101/07-state.md": "stateful-computations.md",
   "jax-101/08-pjit.rst": "sharded-computation.md",
-  "jax-101/index.rst": "tutorials.rst",
+  "jax-101/index.rst": "jax-101.rst",
   "notebooks/external_callbacks.md": "external-callbacks.md",
```

Comment:
Any newly deleted sources should be added to this mapping to prevent broken hyperlinks (e.g. map `advanced_guide` to `advanced_guides`, and `tutorials` and `user_guides` should be mapped to whatever is replacing them).

Reply:
The goal is to create redirects in case there are existing URLs out there that point to the JAX docs. Each time we rename or remove a file, it would otherwise break those links.

Reply:
Thanks! I have now added a redirect for "user_guides.rst" to point now to "advanced_guides.rst" since most of the User Guides were moved there. If that's not correct, please let me know.